eventarc/pubsub/Dockerfile

```diff
@@ -0,0 +1,41 @@
+# Copyright 2020 Google LLC
```

Comment:
If this isn't explicitly for a quickstart tutorial like the audit logs please remove.

Reply:
This is for the Eventarc Pub/Sub quickstart.